openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -28,7 +28,10 @@
 from rest_framework.views import APIView
 
 from common.djangoapps.edxmako.shortcuts import render_to_string
-from openedx.adg.lms.student.helpers import compose_and_send_adg_password_reset_email
+from openedx.adg.lms.student.helpers import (
+    compose_and_send_adg_password_reset_email,
+    send_adg_password_reset_success_email
```

Comment:
Pls update imports. There is an un-used import.

Reply:
No bro that is being used to send reset email. I added the import for reset success email

Reply:
Oooopsss... Sorry. My bad!